File: src/main/resources/static/service/talaba_service.js
var api = "/api/talaba/";
var talaba_service = {
    getAll: function (success, error) {
        return http.get(api, success, error);
    },
    getById: function (id, success, error) {
        return http.get(api + "/" + id, success, error);
    },
    create: function (data, success, error){
        return http.post(api, data, success, error);
    },
    update: function (data, success, error){
         return http.put(api, data, success, error);
    },
    deleteById: function(id, success, error) {
    return http.delete(api + "/" + id, success, error);
    }
}

File: src/main/java/uz/talaba/talabauz/service/TalabaService.java
package uz.talaba.talabauz.service;

import uz.talaba.talabauz.entity.Talaba;

import java.util.List;

public interface TalabaService {
    public List<Talaba> getAll();
    public Talaba getById(Long id);
    public Talaba create(Talaba talaba);
    public Talaba update(Talaba talaba);
    public void delete(Talaba talaba);
    public void deleteById(Long id);


}
